### ISSUE TITLE
Ecto.Rescope.Query? Cannot expand struct Ecto.Query

### ISSUE THREAD

**Issue by janpieper** [erikreedstrom/ecto_rescope#4]
When fetching `ecto_rescope` from hex.pm (`{:ecto_rescope, "~> 0.1.1"}`), the package contains `lib/ecto_rescope/query.ex` with `Ecto.Rescope.Query` which does not compile:

```
== Compilation error in file lib/ecto_rescope/query.ex ==                                                            
** (CompileError) lib/ecto_rescope/query.ex:29: Ecto.Query.__struct__/0 is undefined, cannot expand struct Ecto.Query
    lib/ecto_rescope/query.ex:29: (module)                                                                           
```

After changing to `{:ecto_rescope, github: "erikreedstrom/ecto_rescope", tag: "v0.1.1"}`, everything works fine again :thinking:

**Comment by erikreedstrom** [comment]
Hmm... ok. Will take a look.

**Comment by christianjgreen** [comment]
This should now be fixed with the latest release `v0.1.2`. @janpieper can you confirm?

**Comment by christianjgreen** [comment]
Latest hex package is confirmed working. @janpieper I'll be closing this issue soon :)
